src/oic/utils/aes.py

```diff
@@ -218,8 +220,8 @@ def decrypt_and_verify(self, cipherdata, tag):
     iv_ = os.urandom(16)
     encrypted_msg = encrypt(key_, msg_, iv_)
     txt = decrypt(key_, encrypted_msg, iv_)
-    assert txt == msg_
+    assert txt == msg_  # nosec
 
     encrypted_msg = encrypt(key_, msg_, 0)
     txt = decrypt(key_, encrypted_msg, 0)
-    assert txt == msg_
+    assert txt == msg_  # nosec
```

Comment:
We should probably deprecate (or remove right away) this whole `__main__` part. I do not think there is a real world usage for this?

Reply:
Agreed.